### PR TITLE
test(lint): revert previous assertion

### PR DESCRIPTION
**Pull request by P0lip**
**Checklist**

- [x] Tests added / updated
- [x] Docs added / updated
- [x] Check this off Breaking change?

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

Yeah, I know previous looked a bit different, but it didn't make much sense.
Right now it's slightly better and what's more important consistent for both CRLF and LF line endings.

### CODE REVIEW

File: src/cli/commands/__tests__/lint.test.ts [hunk 1]
```diff
@@ -202,8 +202,7 @@ describe('lint', () => {
         .command(['lint', invalidOas3SpecPath, '-r', standardOas3RulesetPath])
         .it('outputs warnings in default format', ctx => {
           expect(ctx.stdout).toContain(
-            'warning  api-servers       OpenAPI `servers` must be present and non-empty array',
-            // Restore to once Windows new line is sorted '1:5  warning  api-servers       OpenAPI `servers` must be present and non-empty array',
+            '1:1  warning  api-servers       OpenAPI `servers` must be present and non-empty array',
```
Comment (XVincentX) [inline src/cli/commands/__tests__/lint.test.ts]: 1:1? Wasn't it 1:5?

Comment (P0lip) [reply src/cli/commands/__tests__/lint.test.ts]: image
:laughing:

Comment (P0lip) [reply src/cli/commands/__tests__/lint.test.ts]: TFW you try to add a description and he doesn't read it anyway :trollface:

Comment (XVincentX) [reply src/cli/commands/__tests__/lint.test.ts]: Yeah the template is so misleading that seems like nobody uses it and I just skip it.

**Pull request merged**